invenio_records_resources/services/records/params/facets.py

```diff
@@ -16,11 +16,27 @@
 class FacetsParam(ParamInterpreter):
```

Comment:
Name emphasis should be on Interpreter: `FacetsInterpreter` rather than `Params` (these classes interpret the params to give a modified search object). This is in keeping with how they are used: https://github.com/inveniosoftware/invenio-records-resources/pull/99/files#diff-6287c413b00ac98631d623e0329c9f0dR97

Reply:
For me the emphasis is on the param. That's why I named the package ``params`` and went with the suffix ``Param``.